### ISSUE TITLE
Hiding social media account categories does not work properly

### ISSUE THREAD

**Issue by Hexstream** [Hexstream/common-lispers.hexstreamsoft.com#9]
Sorry, (mostly) due to an architectural bug in my homegrown preferences system (I think), unselecting categories like `Coding`, `Microblogging` and `Funding` only updates the UI and does not actually hide the children fields.

For now, the workaround is to unselect the children fields themselves.
For instance, instead of unselecting `Microblogging`, unselect `Twitter` and `Mastodon`.

It could take a while before I get around to fixing this, since this would take some thinking for a proper fix instead of a ugly hack. Sorry about this.

**Comment by lispm** [comment]
Remove my data. It's trivial.

**Comment by Hexstream** [comment]
Please don't spam in unrelated threads.
Please behave yourself like an adult.

**I will absolutely report such completely unrelated and off-topic messages as spam (and perhaps additionally delete them) and your account could be affected.**

**Comment by lispm** [comment]
Your antisocial behavior is not unexpected.

**Comment by Hexstream** [comment]
You are patently insane, and please stop spamming this thread with unrelated content at once.

**Comment by lispm** [comment]
Stop ignoring my request to be deleted.

**Comment by Hexstream** [comment]
I have been highly responsive about it, and I still am.
I will simply not give you the quick shutdown that you are desperately seeking.
(I will also not play this stupid game forever, and I will not let you destabilize this project.)

**Comment by lispm** [comment]
Stop ignoring my request to be deleted.

**Comment by lispm** [comment]
You have destabilized your project from the start, by not asking people whether they want to be part of this. This is doomed to fail.

**Comment by Hexstream** [comment]
> You have destabilized your project from the start, by not asking people whether they want to be part of this.

That's quite incorrect, given the political landscape, it is patently obvious that this project could never even have started if I had started by asking people if they want to be in the list. I believe you are entirely aware of this.

I'm sorry but your insane feud does not destabilize the project in the short term, I am letting you roam free *because* I believe it's worth it to let you exhibit your insanity a bit before eventually being forced to rein you in.

> This is doomed to fail.

If it is, then I don't even know what you're trying to do here. Sit back and let it fail...

> You can't force people to be in YOUR community.

I'm not. Making a list of people does not create a community.
Go ahead and make a list of people, you'll see that it won't create a community...

> I want to be delisted.

Now where have I hear this before...

**Comment by lispm** [comment]
> I don't even know what you're trying to do here

I want to be delisted.

**Comment by Hexstream** [comment]
Your means are demonstrably ineffective.

**Comment by Hexstream** [comment]
Rainer Joswig (lispm) shat/spammed here then [deleted all his messages in shame](https://web.archive.org/web/20190418013357/https://github.com/Hexstream/common-lispers.hexstreamsoft.com/issues/9), I'm not sure what the hell kind of a strategy that is...

(Unfortunately, it seems I was not able to capture his messages in time, so we only have my replies. Sorry.)
